src/main/java/duke/task/ToDo.java

```diff
@@ -0,0 +1,47 @@
+package duke.task;
+
+/**
+ * Class that simulates the todo task that user has inputted.
+ */
+
```

Comment:
I noticed some spacing inconsistencies in some of the files too. You might want to check all files to ensure consistency!

Reply:
thanks!